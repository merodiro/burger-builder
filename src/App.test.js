import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import App from './App'

describe('<App>', () => {
  
  it('renders the same output', () => {
    const component = shallow(<App />)
    
    expect(component).toMatchSnapshot()
  })

})
