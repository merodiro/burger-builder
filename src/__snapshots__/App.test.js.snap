// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> renders the same output 1`] = `
<div>
  <Layout>
    <BurgerBuilder />
  </Layout>
</div>
`;
